### ISSUE TITLE
Where is the source code?

### ISSUE THREAD

**Issue by jacopotediosi** [MFMokbel/Crawlector#1]
This repository contains only the README and LICENSE files.

Should we consider Crawlector to be "closed source" (so it is quite weird to host it on GitHub, which is made for open source software) or do you plan to release the code in the future?

**Comment by MFMokbel** [comment]
Hi @jacopotediosi,

Yes, I'm planning to release the source code sometime in the future. First, will start by releasing the source code of [EKFiddle2Yara](https://github.com/MFMokbel/EKFiddle2Yara) tool.

**Comment by n0tduck1e** [comment]
mate, are still planning on releasing it ? I would love to see how you guys implemented it :)

**Comment by jacopotediosi** [comment]
> mate, are still planning on releasing it ? I would love to see how you guys implemented it :)

Yeah, this project could be a new cybersecurity milestone, a real revolution, but unfortunately the code is still missing 😔

**Comment by MFMokbel** [comment]
Hi all,

Which functionality of Crawlector are you interested in seeing open-sourced?

**Comment by vuln000** [comment]
Hello! 
I am also very interested in the design architecture and implementation details of Crawlector.

**Comment by MFMokbel** [comment]
Hello,

Will open-source it sometime in the next year or end of year this year.

**Comment by data-Ja3g3r** [comment]
Any update if you plan to relase this ?

**Comment by jacopotediosi** [comment]
Hello! Any news on this one?
I was so excited when I saw the presentation of this tool at the NoHat Conference, it's a little sad that the source code was never released.

**Comment by jacopotediosi** [comment]
Hey there, welcome in 2026!
4 years has passed since your talk at NoHat. I feel this is THE year, I really hope source code will be finally released for this so great project!
Meanwhile, greetings from Italy!
